### PR TITLE
PHP7.4: Update doc

### PR DESCRIPTION
**Pull request by pierallard**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,10 +6,15 @@ RUN apt-get update && \
...
         python ssh rsync curl \
         python-jinja2 \
-        python-sphinx \
-        php7.3-apcu php7.3-bcmath php7.3-cli php7.3-curl php7.3-fpm php7.3-gd php7.3-intl php7.3-mysql php7.3-xml php7.3-zip php7.3-mbstring && \
-    echo "memory_limit = 1024M" >> /etc/php/7.3/cli/php.ini && \
-    echo "date.timezone = UTC" >> /etc/php/7.3/cli/php.ini && \
+        python-sphinx && \
+    wget -O /etc/apt/trusted.gpg.d/php.gpg https://packages.sury.org/php/apt.gpg && \
+    sh -c 'echo "deb https://packages.sury.org/php/ buster main" > /etc/apt/sources.list.d/php.list' && \
+    apt-get update && \
```
Comment (nmarniesse) [inline Dockerfile:12]: It's the second *apt-get update* of the command. It would be better to do it only once IMO (so move the source list on top)

**Review by nmarniesse — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (pierallard) [reply Dockerfile:12]: I can't, I need wget before :)

File: cloud_edition/flexibility_mode/docs/partners.rst [hunk 2]
```diff
@@ -11,7 +11,7 @@ Partners
 +----------------------------+--------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
 | ``partners_elasticsearch`` | ``status|start|restart`` | Show status or start/restart elasticsearch daemon                                                                                                |
 +----------------------------+--------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
-| ``partners_php7.3-fpm``    | ``status|start|restart`` | Show status or start/restart php-fpm daemon (Command name can vary depending on php version)                                                     |
+| ``partners_php7.4-fpm``    | ``status|start|restart`` | Show status or start/restart php-fpm daemon (Command name can vary depending on php version)                                                     |
```
Comment (mmetayer) [inline cloud_edition/flexibility_mode/docs/partners.rst:14]: do you know how these commands are generated? do we need to update another repo?

File: install_pim/manual/system_requirements/system_install_ubuntu_1804.rst [hunk 3]
```diff
@@ -36,10 +36,10 @@ Now is the time to install what has been configured in the step before:
 
 When installing MySQL 8.0, you'll have to choose the authentication method. Please select *Use Legacy Authentication Method* as the *Strong Password Encryption* is not yet supported by Akeneo PIM.
 
-PHP 7.3
+PHP 7.4
 *******
 
-As Ubuntu 18.04 only provides PHP 7.2, we need to use `Ondrej Sury <https://deb.sury.org/>`_ packages to install PHP 7.3..
+As Ubuntu 18.04 only provides PHP 7.2, we need to use `Ondrej Sury <https://deb.sury.org/>`_ packages to install PHP 7.4..
```
Comment (mmetayer) [inline install_pim/manual/system_requirements/system_install_ubuntu_1804.rst:42]: we might want to update the doc for a newer version of ubuntu as well (e.g Ubuntu 20.04 LTS), but that's another story...

**Review by mmetayer — APPROVED** [review]

File: cloud_edition/flexibility_mode/docs/partners.rst (hunk 2 above)
Comment (pierallard) [reply cloud_edition/flexibility_mode/docs/partners.rst:14]: I already warned Platypus squad, they will do it.

**Pull request merged**